src/Views/WhatsNewPage.xaml.cs

```diff
@@ -48,6 +55,16 @@ private async void OnLoaded(object sender, Microsoft.UI.Xaml.RoutedEventArgs e)
                 continue;
             }
 
+            // When the Dev Drive feature is not enabled don't show the learn more uri link, but instead move the learn more text into the button content.
+            if (card!.PageKey!.Equals(DevDrivePageKeyUri.AbsoluteUri, StringComparison.OrdinalIgnoreCase))

```

Comment:
Nit: If you're doing string comparison, I think it would be better to do `string.Equals(card?.PageKey, uri, ...)` That handles null already, so you don't have to do it here and risk an exception if it happens to be null

Reply:
updated thanks